tests/common.py

```diff
@@ -397,11 +397,11 @@ def zigpy_device_from_device_data(
                 for attr in cluster["attributes"]:
                     attrid = int(attr["id"], 16)
 
-                    if attr["value"] is not None:
+                    if attr.get("value", None) is not None:
```

Comment:
nit: I don't think we need to specify `None` here, as that should be the default return type for non-existent keys in dictionaries, but we can also keep it if that's more explicit or whatever.